test/guest/cockpit-fedora-22.setup

```diff
@@ -2,6 +2,13 @@
 
 set -ex
 
+# HACK - virt-resize might not be able to resize our xfs rootfs,
+# depending on how it was compiled and which plugins are installed,
+# and will just silently not do it.  So we do it here.
+#
+xfs_growfs /
+df -h /
```

Comment:
I think so. It makes sense to have this when troubleshooting running out of disk space. In fact ```set -x``` is also debug output. The creation of a VM gets to be noisy. It isn't done very often.